src/style.css

```diff
@@ -1 +1,131 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir-Book, sans-serif;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy, sans-serif;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 400;
+}
+
+body {
+  margin: 0;
+}
+
+:root {
+  --width: 100%;
+  --heigth: 70px;
```

Comment:
There is a typo in the CSS variable name '--heigth'. It should be '--height'. This typo could lead to unexpected results or confusion when using this variable.